api/sales_dashboard/views.py

```diff
@@ -56,6 +56,13 @@ def get_queryset(self):
                 Q(name__icontains=search_term) | Q(users__email__icontains=search_term)
             )
 
+        if self.request.GET.get("filter_plan"):
+            filter_plan = self.request.GET["filter_plan"]
+            if filter_plan == "free":
+                queryset = queryset.filter(subscription__isnull=True)
+            else:
+                queryset = queryset.filter(Q(subscription__plan__icontains=filter_plan))
```

Comment:
The `Q` is unnecessary here.

```suggestion
                queryset = queryset.filter(subscription__plan__icontains=filter_plan)
```

Reply:
Also, I don't really like the `icontains`. We should either include all the plan options in the drop down (we could even make that dynamic) or have a dictionary on our side that maps these simplified plan 'groups' to the actual plans.

Reply:
You could also probably shave off some lines of code using a ternary statement. 

Something like: 

```python
filter_args = {"subscription__isnull": True} if filter_plan = "free" else {"subscription__plan__icontains": filter_plan}
queryset = queryset.filter(**filter_args)
```

That might look a bit messy once black gets involved though so not adverse to the `if` / `else` if so.

Reply:
The reason for the `icontain` is that there are a reasonable number of plan ids in Chargebee that we need to match against (`scale-up-annual`, `scale-up-v2` etc) so this was a way of matching against those. We could use a dictionary but that then means API commits when we add a new plan.

Reply:
In addition, this is just to filter roughly so we can spot accounts we want to contact about expansion.

Reply:
Yeah, fair enough. Just a bit skanky. I'm just concerned that we might add a plan in the future like: 

`startup-scale` or something and the filter will get confused. Probably not a concern for now.